server/api/index.js

```diff
@@ -2,6 +2,8 @@ const router = require('express').Router()
 module.exports = router
 
 router.use('/users', require('./users'))
+router.use('/products', require('./product'))
+router.use('/category', require('./category'))
```

Comment:
Might want to consider staying consistent and making this `categories` (but it's not a big deal).